packages/simuwaerm/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/file-acomplaint/simuwaerm"
+TERMUX_PKG_DESCRIPTION="A simple heat transfer simulator in pure Rust"
+TERMUX_PKG_GROUPS="science"
```

Comment:
This package isn't really usable for scientific purposes. Also I don't think it has any practical value either, considering its description on home page.

In my opinion, this package is a candidate for adding to user repository: https://github.com/termux-user-repository/tur. Leaving final decision to other maintainers.